docs/source/index.rst

```diff
@@ -1,15 +1,20 @@
-.. Lumigator documentation master file, created by
-   sphinx-quickstart on Tue Oct 29 13:57:02 2024.
-   You can adapt this file completely to your liking, but it should at least
+.. You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
 Mozilla.ai Lumigator
 ====================
 
 Lumigator is an open-source platform built by Mozilla.ai for guiding users through the process of
-selecting the right language model for their needs. Currently, we support evaluating summarization
+selecting the right language model for their needs.
+
+Currently, we support evaluating summarization
 tasks using sequence-to-sequence models, like BART and BERT, and causal architectures, like GPT and
-Mistral. However, we will be expanding to other machine learning tasks and use-cases.
+Mistral.
+
+These models can either be loaded and run by Lumigator
+(if they are models stored in `Huggingface Hub <https://huggingface.co/models>`_), or the models can be reached through any OpenAIAPI
+compatible interface (for example, llamafile or OpenAI GPT models). Lumigator will soon
+support additional tasks such as translation  (See `#628 <https://github.com/mozilla-ai/lumigator/issues/628>`_).
```

Comment:
@HareeshBahuleyan  this part will need to be updated when you've implemented translation.